generator-prisma-client/nextjs-starter-webpack/README.md

```
@@ -70,26 +70,26 @@
 
 ```
-pnpm prisma migrate dev --name init
+pnpm prisma generate
 ```
 
-You now have an empty `Quotes` table in your database. Next, run the [seed script](./prisma/seed.ts) to create some sample records in the table:
+### 4. Run a migration to create the database structure and seed the database
+
+The [Prisma schema file](./prisma/schema.prisma) contains a single `Quotes` model and a `QuoteKind` enum. You can map this model to the database and create the corresponding `Quotes` table using the following command:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)